src/legacy/TornadoD3.ts

```diff
@@ -284,11 +280,11 @@ export class TornadoD3 {
       // Place the y axis in the middle of the chart
       ?.attr('transform', 'translate(' + (yAxisWidth(axis) + x(0)) + ', ' + margin.top + ' )')
       .call(yAxis);
-    attrs(svg?.selectAll('.y-axis .domain, .y-axis .tick line'), axis.y.style);
-    attrs(svg?.selectAll('.y-axis .tick text'), axis.y.text.style as any);
+    // attrs(svg?.selectAll('.y-axis .domain, .y-axis .tick line'), axis.y.style);
+    // attrs(svg?.selectAll('.y-axis .tick text'), axis.y.text.style as any);
 
-    attrs(svg?.selectAll('.x-axis .domain, .x-axis .tick line'), axis.x.style);
-    attrs(svg?.selectAll('.x-axis .tick text'), axis.x.text.style as any);
+    // attrs(svg?.selectAll('.x-axis .domain, .x-axis .tick line'), axis.x.style);
+    // attrs(svg?.selectAll('.x-axis .tick text'), axis.x.text.style as any);
```

Comment:
Are these intentionally commented out?

Reply:
nope :) I've put them back in now thanks for catching that